samtranslator/model/sam_resources.py

```diff
@@ -179,6 +179,7 @@ class SamFunction(SamResourceMacro):
         "SnapStart": PropertyType(False, IS_DICT),
         "FunctionUrlConfig": PropertyType(False, IS_DICT),
         "RuntimeManagementConfig": PassThroughProperty(False),
+        "TestAuthOverride": PropertyType(False, IS_BOOL),
```

Comment:
Don't need to add this here, this is only for list of SAM function properites

Reply:
Good catch, i'll remove this.